op_builder/gds.py

```diff
@@ -43,8 +43,13 @@ def is_compatible(self, verbose=False):
                 self.warning("Please install torch if trying to pre-compile GDS")
             return False
 
-        CUDA_HOME = torch.utils.cpp_extension.CUDA_HOME
-        CUDA_LIB64 = os.path.join(CUDA_HOME, "lib64")
+        if not self.is_rocm_pytorch():
+            CUDA_HOME = torch.utils.cpp_extension.CUDA_HOME
+            CUDA_LIB64 = os.path.join(CUDA_HOME, "lib64")
+        else:
```

Comment:
Hi @rraminen - I'm not sure this op is supported on AMD currently, should we copy the form done on other ops that rely on things like cutlass that aren't supported currently?

Reply:
Hi @loadams, I was just thinking the same. Introducing "[WARNING]  gds is not compatible with ROCM" in the is_compatible(). Shall I go ahead and make the change ?

Reply:
Yes that seems reasonable to me!